phoebe/solverbackends/solverbackends.py

```diff
@@ -2101,7 +2094,7 @@ def run_worker(self, b, solver, compute, **kwargs):
                     v = v.to(fitted_units[index]).value
                 p0[p_ind] = v
             else:
-                logger.warning("ignoring {}={} in initial_values as was not found in fit_parameters".format(twig, value))
+                logger.warning("ignoring {}={} in initial_values as was not found in fit_parameters".format(twig_orig, value))
```

Comment:
minor logger fix that only affects cases where fit_parameters include an index (so mainly ld_coeffs).  The logger message will now contain the indices.